### DISCUSSION TITLE
Stopped at 'this->validate()' line when all field validations didn't produce error.

### DISCUSSION THREAD

**Discussion by Idan-Garay** [livewire/livewire#2837] [1. Help] [answered]
Hello Everyone, I've spent so much time with this and I'm still stuck with this problem of not being able to save my field data to database because execution stopped at the 'this->validate()' line (line 64).

Goal:
When Submit button is clicked, it will push through the 'this->validate()' line and create a row for the post in the database

1. Inputted all fields with values
2. Clicked submit
3. Nothing happens aside from input background getting shaded and then back to white color.

livewire component:
<img width="1191" alt="image" src="https://user-images.githubusercontent.com/47768819/117465883-b6105680-af84-11eb-8522-f2cae0ceff41.png">
<img width="869" alt="image" src="https://user-images.githubusercontent.com/47768819/117467641-7480ab00-af86-11eb-92d0-f7ffb85a266f.png">

Livewire Component Handler
<img width="985" alt="image" src="https://user-images.githubusercontent.com/47768819/117466210-07b8e100-af85-11eb-98ab-6872d7e4fb45.png">

Form: 
[Filled with valid data]
<img width="748" alt="image" src="https://user-images.githubusercontent.com/47768819/117466607-7e55de80-af85-11eb-8aca-7321389c5822.png">

[When submit button is clicked. It should prompt the dd but it didn't and just went back to normal]
<img width="673" alt="image" src="https://user-images.githubusercontent.com/47768819/117466814-b2310400-af85-11eb-9031-c5ef989dd429.png">

**Comment by drbornot** [comment]
All that blade code is inside a parent div container??? Can you see in console any issue or in network tab if any request is done after submit???

**Reply by Idan-Garay** [reply]
Sorry for the late reply.

This is before clicking the submit button.
![image](https://user-images.githubusercontent.com/47768819/117524115-ba715980-afee-11eb-9c85-c9b8170d2289.png)

After clicking the submit button.
![image](https://user-images.githubusercontent.com/47768819/117524140-d7a62800-afee-11eb-9356-70d4a1d23d25.png)

![image](https://user-images.githubusercontent.com/47768819/117524164-f3113300-afee-11eb-9512-1c784793a96a.png)

**Comment by Idan-Garay** [comment, marked as answer]
I've found the error and it's because in the rules, I had redundant max in line 40.
![image](https://user-images.githubusercontent.com/47768819/117525155-851b3a80-aff3-11eb-86c1-9e02382e2190.png)

which didn't pass through $this->validate() and I don't know the specific reason why it didn't show any invalid error but I fixed it with this
![image](https://user-images.githubusercontent.com/47768819/117525221-cca1c680-aff3-11eb-9f71-1353ad464094.png)

Now: 
When Submit button is clicked, it pushed through the 'this->validate()' line and created a row for the post in the database
